### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by diomed**
added 2 UI libraries
removed few dead and outdated [archived] links| Name                 | Link                 |
| -------------------- | -------------------- |
| _WindUI_ | _(https://wind-ui.com)_ |
|_PinesUI_  |_(https://devdojo.com/pines)_|

---

- [x] My item is in the right category
- [x] My item is logically grouped below similar items
- [x] My item's name and description respects the conventions of the list
- [x] My item is awesome
- [x] My item is in line with the [Tailwind brand usage guidelines](https://tailwindcss.com/brand#usage)
- [x] I have read and followed the [contribution guidelines](https://github.com/aniftyco/awesome-tailwindcss/blob/master/.github/CONTRIBUTING.md)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -301,23 +300,24 @@
...
+- 🧩 [Layouts for Tailwind](https://layoutsfortailwind.lalokalabs.dev) - Layouts and UI Patterns for Tailwind CSS.
+- 🧩 [HyperUI](https://hyperui.dev) - Open source marketing and ecommerce Tailwind CSS components.
+- 🧩 [Snippets](https://snippets.alexandru.so) - Open source collection of animation snippets made for Tailwind CSS.
+- 🧩 [Fancy Tailwind](https://fancytailwind.com) - Large collection of Tailwind CSS UI components (700+).
+- 🧩 [Myna UI](https://mynaui.com) - Open source UI Components and Marketing Elements made with Tailwind CSS.
 - 🧩 [Sira UI](https://sira.riccox.com) - Customizable and accessible design system which provides TailwindCSS component class name library to build modern UI.
-- 🧩 [RippleUI](https://www.ripple-ui.com/) - Clean, modern and beautiful Tailwind CSS components.
+- 🧩 [RippleUI](https://www.ripple-ui.com) - Clean, modern and beautiful Tailwind CSS components.
+- 🧩 [Wind UI](https://wind-ui.com) - Expertly made, responsive, accessible components in React and HTML ready to be used on your website or app.
+- 🧩 [Pines UI](https://devdojo.com/pines) - Alpine and Tailwind CSS UI library
```
Comment (joshmanders) [inline README.md:312]:
```suggestion
- 🧩 [Pines UI](https://devdojo.com/pines) - Alpine and Tailwind CSS UI library.
```

File: README.md [hunk 2]
```diff
@@ -381,11 +381,11 @@
 - 🚀 [AstroWind](https://github.com/onwidget/astrowind) - Production ready and SEO-friendly template to start a website using Astro and Tailwind CSS.
 - 🚀 [Angular-Tailwind](https://github.com/luciano-work/angular-tailwind) - Dashboard starter kit using Angular and Tailwind CSS.
 - 🚀 [Vue-Resume](https://github.com/luciano-work/vue-resume) - Developer resume template with Tailwind CSS and Vue.
+- 🚀 [Tailwind sites](https://tailwindsites.com) - Free Tailwind CSS site templates for your next project
```
Comment (joshmanders) [inline README.md:384]:
```suggestion
- 🚀 [Tailwind sites](https://tailwindsites.com) - Free Tailwind CSS site templates for your next project.
```

**Review by joshmanders — CHANGES_REQUESTED** [review]
Other than these I don't see anything wrong. What do you think @innocenzi?

**Review by innocenzi — APPROVED** [review]
LGTM after the changes asked by @joshmanders 👍

**Review by joshmanders — APPROVED** [review]

**Pull request merged**